libcontainer/cgroups/systemd/v2.go

```diff
@@ -64,8 +64,7 @@ func unifiedResToSystemdProps(cm *dbusConnManager, res map[string]string) (props
 		if strings.Contains(k, "/") {
 			return nil, fmt.Errorf("unified resource %q must be a file name (no slashes)", k)
 		}
-		sk := strings.SplitN(k, ".", 2)
-		if len(sk) != 2 {
+		if strings.IndexByte(k, '.') < 2 {
```

Comment:
why less than 2? `< 1` should work, right? (Assuming there is a hypothetical one character controller in the future..)

Reply:
You're right, and I have a correct description in the commit message, but not in the code :)

Fixed